code/__defines/callbacks.dm

```diff
@@ -0,0 +1,3 @@
+#define GLOBAL_PROC "some_magic_bullshit"
```

Comment:
is this a debugging thing? doesnt seem like it should be in live code

Reply:
You may refer here https://github.com/tgstation/tgstation/blob/master/code/__DEFINES/callbacks.dm#L1 and probably to any other codebase which uses their callbacks :), of course i could rename this, but i prefer identical stuff from codebase to codebase if possible. And no, this isn't debugging things, its for CALLBACK library, for example you may check QDEL_IN macro in this PR which calls qdel() which is a global proc, that's where it comes in effect. So in short, this needed for globally defined proc calls within callbacks.